app/controllers/catalog_controller.rb

```diff
@@ -227,7 +227,7 @@ def self.modified_field
       all_names = config.show_fields.values.map(&:field).join(' ')
       title_name = solr_name('title', :stored_searchable)
       field.solr_parameters = {
-        qf: "#{all_names} #{title_name} license_label_tesim file_format_tesim all_text_timv",
+        qf: "#{all_names} #{title_name} abstract_tesim license_label_tesim file_format_tesim form_of_work_label_tesim all_text_timv",
```

Comment:
abstract and form_of_work should already be searchable? Are they indexed specially or something?